### PR TITLE
Add cache of zips in home directory

### PR DESCRIPTION
**Pull request by RnbWd**
Added cache folder in home directory that prevents unnecessary downloads

### CODE REVIEW

**Comment by RnbWd** [comment]
adding the cache turned out to be fairly simple:

- download zips in (hidden) home-dir (instead of local dir)
- unzip into local dist (same behavior)
- if used by another library, check if the version already exists, if so, just zip it, no download

what i learned doing this?

- it's (probably) a bad idea to copy open electron apps between projects - errors happened

- zips are safe (I think). saving them saves bandwidth and facilitates offline usage for libs that have eb as a dep or you'd like to use it independent of cli

File: install.js [hunk 1]
```diff
@@ -22,12 +24,28 @@ var paths = {
   win32: path.join(__dirname, './dist/electron.exe')
 }
 
+var cache = {
+  darwin: path.join(getHomePath, './.electron'),
+  linux: path.join(getHomePath, './.electron'),
+  win32: path.join(getHomePath, './.electron')
+}
```
Comment (mafintosh) [inline install.js]: why do we need a different cache for each platform?

**Comment by mafintosh** [comment]
@RnbWd wow thanks! except for my comment this is LGTM

File: install.js (hunk 1 above)
Comment (RnbWd) [inline install.js]: Oh, I suppose we don't. It's a relic from my first attempt at saving the entire file.

**Comment by mafintosh** [comment]
Thanks! Published in 0.25.2-1

**Pull request merged**